ops/global/storage.tf

```diff
@@ -0,0 +1,48 @@
+resource "azurerm_storage_account" "app" {
+  account_replication_type         = "GRS" # Cross-regional redundancy
+  account_tier                     = "Standard"
+  account_kind                     = "StorageV2"
+  name                             = "${local.team}${local.project}${local.env}sa"
+  resource_group_name              = local.resource_group_name
+  location                         = local.location
+  enable_https_traffic_only        = true
+  min_tls_version                  = "TLS1_2"
+  allow_nested_items_to_be_public  = false
+  cross_tenant_replication_enabled = false
+}
+
+resource "azurerm_storage_share" "repository" {
+  name                 = "repository"
+  storage_account_name = azurerm_storage_account.app.name
+  quota                = 5
```

Comment:
I haven't used storage shares. If this 5GB quota is changed to decrease or increase storage, does it recreate this object or update it? Is there the potential to delete data?

Reply:
The documentation indicates that there will be an update, with no potential to delete data. We can work with additional data safety as part of #62, if desired!